Makefile

```diff
@@ -1,7 +1,13 @@
 ROOT:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-IMAGE_NAME:=jenkins4eval/jnlp-slave:test
+IMAGE_NAME:=jenkins4eval/jnlp-slave
 
 build:
-	docker build -t ${IMAGE_NAME} .
-	docker build -t ${IMAGE_NAME}-alpine -f Dockerfile-alpine .
-	docker build -t ${IMAGE_NAME}-jdk11 -f Dockerfile-jdk11 . 
+	docker build -t ${IMAGE_NAME}:latest .
```

Comment:
Ideally, I think this should actually be split like we have on the master docker image Makefile. Being able to build a single platform is nice.

Reply:
Updated.